server/php-runner.cpp

```diff
@@ -80,13 +81,24 @@ void send_slow_net_event_stats(const net_event_t &event, double time_sec) noexce
 
 } // namespace
 
-void PhpScript::error(const char *error_message, script_error_t error_type) noexcept {
+void PhpScript::error(const char *error_message, script_error_t error_type, [[maybe_unused]] const std::optional<int> &triggered_by_signal) noexcept {
   assert (in_script_context == true);
   in_script_context = false;
   current_script->state = run_state_t::error;
   current_script->error_message = error_message;
   current_script->error_type = error_type;
-  stack_end = reinterpret_cast<char *>(exit_context.uc_stack.ss_sp) + exit_context.uc_stack.ss_size;
+  stack_end = reinterpret_cast<char *>(get_context_stack_ptr_portable(exit_context)) + get_context_stack_size_portable(exit_context);
+
+#if defined(__linux__) && defined(__x86_64__)
+  // The PhpScript::error may be produced in process of signal handling. The default behavior on Linux-based platforms
```

Comment:
We also have own implementation for `darwin aarch64`. Do we need same logic for this?

Reply:
I will check it. Previously, we also had own implementation for `darwin aarch64` but no needed a special handling for this case. I may be a result of different signal subsystems.

Reply:
Checked, thanks for notice, fixed for `darwin aarch64`